### ISSUE TITLE
Why the same point cloud frame calculation result is not 0?

### ISSUE THREAD

**Issue by S-kewen** [FengZicai/shape-measure#2]
I tried to input p1 and p2 as the same point cloud frame, but the result is not 0

here is my code:

dist01 = EMDLoss()
print('compute EMDLoss ALG1')
p1 = torch.rand(32, 1024, 3).cuda()#.double()
p2 = p1

p1.requires_grad = True
p2.requires_grad = True

s = time.time()

cost  = dist01(p1, p2)
print('Wasserstein (EMD) cost from ALG1:')
print(cost)

loss1 = torch.sum(cost)
print('Wasserstein (EMD) loss from ALG1: %.05f'%loss1)
